src/services/deckService.js

```diff
@@ -19,14 +19,65 @@ class DeckService {
     await sheet.loadCells("H4:H520");
     const cells = [];
     for (let i = 4; i < 100; i++) {
-      cells.push(sheet.getCellByA1(`H${i}`));
+      //cells.push(sheet.getCellByA1(`H${i}`));
+      if (!sheet.getCellByA1(`H${i}`)._rawData.formattedValue) {
+        console.log(i-1);
+        return i;
+      }
```

Comment:
Watch out using properties which names start with an underscore (`._rawData` in this case). Those are supposed to be private and we shouldn't use them.

https://stackoverflow.com/questions/4484424/underscore-prefix-for-property-and-method-names-in-javascript.

Also what is this code supposed to accomplish?